dags/common/cleanup.py

```diff
@@ -64,11 +68,9 @@ def remove_unnecessary_fields(obj):
 
 
 def remove_orcid_prefix(obj):
-    orcid_prefixes = ("orcid:", "https://orcid.org/", "http://orcid.org/")
+    pattern = re.compile(r"https{0,1}://orcid.org/|ORCID-|orcid:", flags=re.I)
```

Comment:
you added a flag `ignorecase` so maybe you can put the whole pattern in lowercase